src/pages/channels/ChannelView.tsx

```diff
@@ -0,0 +1,861 @@
+            ...loginRequest,
+            account: this.context.accounts[0]
+        })
+        .then((response: any) => {
+            apiClient.getBrokersPsp({
+                Authorization: `Bearer ${response.idToken}`,
+                ApiKey: "",
+                page: 0,
+                limit,
+                code
+            }).then((resp: any) => {
+                if (resp.right.status === 200) {
+                    const items: Array<any> = [];
+                    resp.right.value.brokers_psp.map((broker_psp: any) => {
+                        // eslint-disable-next-line functional/immutable-data
+                        items.push({
+                            value: broker_psp.broker_psp_code,
+                            label: broker_psp.broker_psp_code,
+                        });
+                    });
+                    callback(items);
+                } else {
+                    callback([]);
+                }
+            }).catch(() => {
+                toast.error("Operazione non avvenuta a causa di un errore", {theme: "colored"});
+                callback([]);
+            });
+        });
+    }
+
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    promiseWfespOptions(inputValue: string, callback: any) {
+        this.context.instance.acquireTokenSilent({
+            ...loginRequest,
+            account: this.context.accounts[0]
+        })
+            .then((response: any) => {
+                apiClient.getWfespPlugins({
+                    Authorization: `Bearer ${response.idToken}`,
+                    ApiKey: "",
+                }).then((resp: any) => {
+                    if (resp.right.status === 200) {
+                        const items: Array<any> = [];
+                        // eslint-disable-next-line functional/immutable-data
+                        items.push({
+                            value: '-',
+                            label: '-',
+                        });
+                        resp.right.value.wfesp_plugin_confs.map((plugin: any) => {
+                            // eslint-disable-next-line functional/immutable-data
+                            items.push({
+                                value: plugin.id_serv_plugin,
+                                label: plugin.id_serv_plugin,
+                            });
+                        });
+                        callback(items);
+                    } else {
+                        callback([]);
+                    }
+                }).catch(() => {
+                    toast.error("Operazione non avvenuta a causa di un errore", {theme: "colored"});
+                    callback([]);
+                });
+            });
+    }
+
+    handleWfespChange(event: any) {
+        const channel: ChannelDetails = this.props.channel;
+        // eslint-disable-next-line functional/no-let
+        let value = event.value;
+        if (value === 'null') {
```

Comment:
could you check if should be `'-'`?

Reply:
Yes, but when changing the value at line 195 we have to choose between:
- value = null; => defined to undefined error but correct operation
- value = '-' => does not work because payment type '-' does not exist